nio_cli/commands/config.py

```diff
@@ -140,25 +141,42 @@ def success_message(name, ssl, self_signed, niohost, nioport, standalone_pubkeep
 
     print('')
     print('\033[92m' + 'Success!' + '\033[0m')
+    print('First, start your instance:')
+    print('- Enter your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m')
 
     if (standalone_pubkeeper and nioport <= 1024):
-        print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'sudo niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon: '  + '\033[94m' + 'sudo niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon inn the background: '  + '\033[94m' + 'sudo nohup niod -s nio.conf -s pk_server.conf 2>&1 > /dev/null &' + '\033[0m')
```

Comment:
Typo `inn`. Do we want these docs here? I feel like having them on docs.n.io is enough. And if we have them here we'll have to remember to change them here.

Reply:
I want to avoid the error I saw (and have encountered myself), where one:
- starts the service with '&'
- closes their laptop, and then 
- gets a port conflict message when trying to start it again.